Utils/Pieces/Queen.py

```diff
@@ -1,4 +1,6 @@
 from Utils.Piece import Piece
+from .Bishop import Bishop
+from .Rook import Rook
 class Queen(Piece):
```

Comment:
Lint it properly. No `enter` after imports, when you declare a class should be punishable by law ;p